From 3aac9eaf8c20a4611c7ad2495c8a99e19f153ae3 Mon Sep 17 00:00:00 2001
From: Jeremy Kerr <jk@ozlabs.org>
Date: Wed, 20 Nov 2019 12:32:02 +0800
Subject: [PATCH 4/9] lib/types: consolidate struct system_info layout with
 serialised version

We currently serialise the BMC MAC at the end of a system info message,
so update struct system_info to suit.

Signed-off-by: Jeremy Kerr <jk@ozlabs.org>
---
 lib/types/types.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/types/types.h b/lib/types/types.h
index 433a37b..22fa337 100644
--- a/lib/types/types.h
+++ b/lib/types/types.h
@@ -133,11 +133,11 @@ struct system_info {
 	char			**bmc_golden;
 	unsigned int		n_bmc_current;
 	unsigned int		n_bmc_golden;
-	uint8_t			*bmc_mac;
 	struct interface_info	**interfaces;
 	unsigned int		n_interfaces;
 	struct blockdev_info	**blockdevs;
 	unsigned int		n_blockdevs;
+	uint8_t			*bmc_mac;
 };
 
 #define HWADDR_SIZE	6
-- 
2.17.1

